### ISSUE TITLE
"undefined property mathsdisplay" on the STACK setup screen while installing

### ISSUE THREAD

**Issue by christianp** [maths/moodle-qtype_stack#461]
I've just set up a fresh moodle with mdk, so all debug messages are turned on. When running the plugin upgrade script, I get a lot of warning messages

``` 
Notice: Undefined property: stdClass::$mathsdisplay in /home/christian/moodles/stable_36/moodle/question/type/stack/stack/mathsoutput/mathsoutput.class.php on line 100
```

It looks like this setting is about to be set when I save some settings, but it would be nice to avoid this error.

**Comment by christianp** [comment]
I should've paid attention to GitHub showing me related issues. This was supposedly fixed in #64 in 2013!

**Comment by sangwinc** [comment]
Thank you Christian, much appreciated.
